library/core/src/ops/control_flow.rs

```diff
@@ -79,7 +79,7 @@ use crate::{convert, ops};
 /// [`Break`]: ControlFlow::Break
 /// [`Continue`]: ControlFlow::Continue
 #[stable(feature = "control_flow_enum_type", since = "1.55.0")]
-#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
```

Comment:
`Ord`/`PartialOrd` were intentionally left out as described in https://rust-lang.github.io/rfcs/3058-try-trait-v2.html#traits-for-controlflow

I'd be inclined to leave it like that, unless you can give a good reason why one variant is less than the other.

`Eq`/`Hash` seem fine to add, though.